.github/workflows/style.yml

```diff
@@ -27,6 +27,7 @@ jobs:
 
     - name: Install dependencies
       run: |
+        pip install .
```

Comment:
@xjules , it seems your comment here has been lost. This line is added in order to let `pylint` check that everything is importable, so this is used only to get the dependencies in, not to get komodo itself in.